config/vmdb.tmpl.yml

```diff
@@ -306,20 +306,25 @@ workers:
           :nova: "notifications.*"
           :cinder: "notifications.*"
           :glance: "notifications.*"
-          :neutron: "notifications.*"
-          :quantum: "notifications.*"
```

Comment:
Modifying the config like this is going to require a data migration to migrate configs stored in the configurations table.